### PR TITLE
add docs on setting localstack on kubernetes

### PR DESCRIPTION
**Pull request by HarshCasper**

### CODE REVIEW

**Review by whummer — APPROVED** [review]
LGTM, 👍 only a few minor comments/suggestions. Would be great to get another pair of eyes from @alexrashed 👀 who is the code owner of our Helm charts.

File: content/en/references/localstack-on-kubernetes.md
Comment (HarshCasper) [reply content/en/references/localstack-on-kubernetes.md:25]: Yeah!

**Pull request merged**
